autoclicker.py

```diff
@@ -1,22 +1,31 @@
 import pyautogui
 from pynput.keyboard import *
+from random import randrange
 
 #  ======== settings ========
```

Comment:
Clean up grammar in comments

Reply:
Addressed in [8a098de](https://github.com/isaychris/python-autoclicker/pull/8/commits/8a098de6a168325f96ace73b85c213575c76c2b2)